code/widgets/org.eclipse.scout.jswidgets.ui.html/src/main/js/jswidgets/table/ColumnPropertiesBox.json

```diff
@@ -134,6 +141,12 @@
       "label": "Text",
       "enabled": false
     },
+    {
+      "id": "TooltipTextField",
+      "objectType": "StringField",
+      "label": "Tooltip Text",
```

Comment:
Please always use the actual property name, in this case Header Tooltip Text instead of just Tooltip Text.

Reply:
is renamed now, thanks